tool_commands/guess_indicators/test_guess_indicators.py

```diff
@@ -0,0 +1,238 @@
+from ThreeDiToolbox.sql_models.constants import Constants
+
+import shutil
+import pytest
+from pathlib import Path
+
+
+def get_pump_or_outlet_manholes(session, ids):
+    """
+    :param ids: [list] this contains ids from either manhole_outlet or manhole_pump
+    :return: a list with tuples (manhole.id, manhole.manhole_indicator)
+    """
+    manholes_id_indicator = {}
+    # only return manhole
+    sql_manholes = (
+        session.query(Manhole)
+        .join(Manhole.connection_node)
+        .filter(ConnectionNode.id.in_(ids))
+    )
+    for manhole in sql_manholes:
+        # fill dict
+        manholes_id_indicator[manhole.id] = manhole.manhole_indicator
+    return manholes_id_indicator
+
+
+def get_all_manholes(session):
+    """ get all manholes from sqlite (also manhole_outlet, manhole_pump)
+    :return: a list with tuples (manhole.id, manhole.manhole_indicator) """
+    manholes_id_indicator = {}
+    sql_manholes = session.query(Manhole)
+    for manhole in sql_manholes:
+        # fill dict
+        manholes_id_indicator[manhole.id] = manhole.manhole_indicator
+    return manholes_id_indicator
+
+
+def get_manholes(session, outlet_bool=False, pump_bool=False, all_bool=False):
+    """ get manholes from sqlite: OR only manhole_outlet, OR only manhole_pumps, OR
+    all manholes (manhole_outlet + manhole_pump + other manholes)
+    :return: a list with tuples (manhole.id, manhole.manhole_indicator)  """
+    # only 1 of kwargs may be True
+    assert sum([outlet_bool, pump_bool, all_bool]) == 1
+    if pump_bool:
+        ids = session.query(Pumpstation.connection_node_start_id)
+    elif outlet_bool:
+        ids = session.query(BoundaryCondition1D.connection_node_id)
+    if pump_bool or outlet_bool:
+        manholes_id_indicator = get_pump_or_outlet_manholes(session, ids)
+    if all_bool:
+        manholes_id_indicator = get_all_manholes(session)
+    return manholes_id_indicator
+
+
+def get_all_pipes(session):
+    pipes = {}
+    # only return manhole
+    sql_pipes = session.query(Pipe)
+    for pipe in sql_pipes:
+        # fill dict
+        pipes[pipe.id] = (pipe.friction_type, pipe.friction_value, pipe.material)
+    return pipes
+
+
+# def get_all_pipes_friction_type(session):
+#     pipes_id_friction_type = {}
+#     # only return manhole
+#     sql_pipes = session.query(Pipe)
+#     for pipe in sql_pipes:
+#         # fill dict
+#         pipes_id_friction_type[pipe.id] = pipe.friction_type
+#     return pipes_id_friction_type
+#
+#
+# def get_all_pipes_friction_value(session):
+#     pipes_id_friction_value = {}
+#     # only return manhole
+#     sql_pipes = session.query(Pipe)
+#     for pipe in sql_pipes:
+#         # fill dict
+#         pipes_id_friction_value[pipe.id] = pipe.friction_value
+#     return pipes_id_friction_value
+
+
+def empty_manhole_indicator(session, manholes_pre_empty):
+    """ empty column manhole_indicator (set is to NULL) of table v2_manhole """
```

Comment:
```suggestion
    """Empty column manhole_indicator (set is to NULL) of table v2_manhole."""
```

Geen spaties om de tekst heen. En zin met een hoofdletter beginnen.

De punt aan het eind van die eerste zin van een docstring: daar ben ik zelf niet consequent in. Tegenwoordig doe ik het wel, vroeger nooit. Geloof ik.